WindowsServerDocs/remote/remote-desktop-services/clients/change-listening-port.md

```diff
@@ -9,7 +9,7 @@ ms.localizationpriority: medium
 ---
 # Change the listening port for Remote Desktop on your computer
 
->Applies to: Windows 10, Windows 8.1, Windows 8, Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2008 R2
+> Applies to: Windows 10, Windows 8.1, Windows 8, Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2008 R2, Windows Server 2019
```

Comment:
"Windows Server 2019" is duplicated on this row

```suggestion
> Applies to: Windows 10, Windows 8.1, Windows 8, Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2008 R2
```

Reply:
Thank you for noticing and reporting. 👍🏻